app/views/rooms/show.html.erb

```diff
@@ -1,18 +1,41 @@
-<%= render @room %>
+<div class="py-3">
+  <div class="d-flex flex-row justify-content-between align-items-center mb-3">
+    <h1>Room: <%= @room.full_name  %></h1>
+    <div class="d-flex flex-column gap-1">
+      <%= link_to 'Edit Room', edit_room_path(@room), class: 'btn btn-primary' %>
+      <%= link_to room_path(@room), class: 'btn btn-danger',
+                  data: { turbo_confirm: "Are you sure you want to Archive this Room?" , turbo_method: :delete } do %>
```

Comment:
Let's not remove rooms with the Archive button. For now, you can keep the link path, but please remove `data: { turbo_confirm: "Are you sure you want to Archive this Room?" , turbo_method: :delete }`